spec/tags/truffle/launcher_tags.txt

```diff
@@ -44,23 +44,14 @@ aot:The launcher '-J:cp ' adds the jar
 aot:The launcher '--jvm.classpath=' adds the jar
 aot:The launcher '--jvm.cp=' adds the jar
 slow:The launcher enables deterministic hashing if -Xhashing.deterministic is set
-slow:The launcher 'gem' runs when symlinked
-slow:The launcher 'irb' runs when symlinked
-slow:The launcher 'rake' runs when symlinked
-slow:The launcher 'rdoc' runs when symlinked
-slow:The launcher 'ri' runs when symlinked
-slow:The launcher 'ruby' runs when symlinked
-slow:The launcher 'testrb' runs when symlinked
-slow:The launcher 'truffleruby' runs when symlinked
-aot:The launcher 'gem' runs when symlinked
-aot:The launcher 'irb' runs when symlinked
-aot:The launcher 'rake' runs when symlinked
-aot:The launcher 'rdoc' runs when symlinked
-aot:The launcher 'ri' runs when symlinked
-aot:The launcher 'ruby' runs when symlinked
-aot:The launcher 'testrb' runs when symlinked
-aot:The launcher 'truffleruby' runs when symlinked
-slow:The launcher 'truffleruby' in 'bin' runs when symlinked
+aot:The launcher 'gem' in 'bin' directory runs when symlinked
+aot:The launcher 'irb' in 'bin' directory runs when symlinked
+aot:The launcher 'rake' in 'bin' directory runs when symlinked
+aot:The launcher 'rdoc' in 'bin' directory runs when symlinked
+aot:The launcher 'ri' in 'bin' directory runs when symlinked
+aot:The launcher 'ruby' in 'bin' directory runs when symlinked
+aot:The launcher 'testrb' in 'bin' directory runs when symlinked
+aot:The launcher 'truffleruby' in 'bin' directory runs when symlinked
```

Comment:
We still need the slow tags. They can be added by running `jt test fast spec/truffle/launcher_spec.rb`.

Reply:
they are bellow

Reply:
This seems to remove
```
slow:The launcher 'truffleruby' in 'bin' runs when symlinked
```
though.

Reply:
It was replaced before by `slow:The launcher 'truffleruby' in 'bin' directory runs when symlinked`